A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.html

```diff
@@ -13,7 +13,7 @@
             </g>
             <g id="sts-reminder">
                 <rect x="350" y="572" width="50" height="20" fill="black" stroke="white" stroke-width="2" />
-                <text x="375" y="588.5" fill="white" font-size="20px" text-anchor="middle">STS</text>
+                <text x="375" y="588.5" fill="white" style="font-size: 12px;" text-anchor="middle">STS</text>
```

Comment:
Did you test your changes? `pt` and `px` are not equivalent, so I'd be surprised if the `12pt` you had earlier is the same size as the `12px` you have now.

Reply:
![image](https://user-images.githubusercontent.com/23118756/109384335-73e8ea00-792f-11eb-9113-e94f999affce.png)
changed that :) looks good?